i18n-guidelines.md

```diff
@@ -1,24 +1,78 @@
-# 🌐 i18n Guide
+# 🌐 i18n Guidelines
 
-At the moment, we have the course in the following languages:  
+Welcome to our i18n Guidelines! We appreciate your interest in translating our Intro to Open Source course.
 
-* Brazilian Portuguese
-* English
-* French
+At the moment, we have the course in the following languages:
 
-## How can I contribute?
+- English
+- [French](https://intro.opensauced.pizza/#/translations/fr/README)
+- Brazilian Portuguese — work in progress
 
-* **Translate our README**. There are two methods of translation contributions we accept.
-  * **Official Translations**: Official translations start as a post in our discussion board. If there is enough interest and volunteers, we can add the official translation as an option to view within our README.
-  * **Community Translations**: We can’t always support the maintenance of translations. However, we do know that there are contributors out there willing to translate. For that reason, we have our Community Translations section. If you’ve forked the repository and created a translation, you can add a link to your forked translation to `community-translations.md`. We encourage you to also add it to the discussion board in the event it becomes popular enough and has enough support to be moved to an official translation.
-* **Review PRs and translations.** New to reviewing GitHub PRs and translations? No worries, we have you covered in our [Reviewer Process Guide](#reviewer-process-guide).
+## How Can I Contribute?
+
+- **Translate our README**. There are two methods of translation contributions we accept.
+  - **Official Translations**: Official translations start as a post in our discussion board. If there is enough interest and volunteers, we can add the official translation as an option to view within our README.
+  - **Community Translations**: We can’t always support the maintenance of translations. However, we do know that there are contributors out there willing to translate. For that reason, we have our Community Translations section. If you’ve forked the repository and created a translation, you can add a link to your forked translation to `community-translations.md`. We encourage you to also add it to the discussion board in the event it becomes popular enough and has enough support to be moved to an official translation.
+- **Review PRs and translations.** New to reviewing GitHub PRs and translations? No worries, we have you covered in our [Reviewer Process Guide](#reviewer-process-guide).
```

Comment:
As there are changes in this file, we might as well want to restructure this section as below:

```suggestion
## How Can I Contribute?

There are two types of contributions that we accept:

1. Translate our Intro to Open Source course
2. Review PRs and translations

### Translate the Intro to Open Source Course

We have two types of translations:

#### 1. Official Translations

Official translations start as a post in our discussion board. If there is enough interest and volunteers, we can add the official translation as an option to view within our README.

#### 2. Community Translations

We can’t always support the maintenance of translations. However, we do know that there are contributors out there willing to translate. For that reason, we have our Community Translations section. If you’ve forked the repository and created a translation, you can add a link to your forked translation to `community-translations.md`. We encourage you to also add it to the discussion board in the event it becomes popular enough and has enough support to be moved to an official translation.

### Review PRs and Translations

If you are familiar with the translated language(s), you can help us review the translations and the PRs.

Are you new to reviewing GitHub PRs and translations? No worries! We  have you covered in our [Reviewer Process Guide](#reviewer-process-guide).
```

Reply:
I like this version @adiati98z. We just need to hyperlink where it says *discussion board** so contributors can find it.